doc/ref_model/chapters/chapter07.md

```diff
@@ -507,3 +519,190 @@ Table 7-2 shows security capabilities
+| sec.sys.011 | The Platform **must not** reuse the same authentication key-pair (for example, on different hosts, for different services) |  | |
+| sec.sys.012 | The Platform **must** only use secrets encrypted using strong encryption techniques, and stored externally from the component |  | e.g., Barbican (OpenStack) |
+| sec.sys.013 | The Platform **must** provide secrets dynamically as and when needed |  | |
+
+<a name="7.11.3"></a>
+### 7.11.3. Confidentiality and Integrity
+
+| Ref | Requirement | Unit | Definition |
+|---|----|---|----|
+| sec.ci.001 | The Platform **must** support Confidentiality and Integrity of data at rest and in-transit |  | |
+| sec.ci.002 | The Platform **should** support self-encrypting storage devices |  | |
+| sec.ci.003 | The Platform **must** support Confidentiality and Integrity of data related metadata |  | |
+| sec.ci.004 | The Platform **must** support Confidentiality of processes and restrict information sharing with only the process owner (e.g., tenant). |  | |
+| sec.ci.005 | The Platform **must** support Confidentiality and Integrity of process-related metadata and restrict information sharing with only the process owner (e.g., tenant). |  | |
+| sec.ci.006 | The Platform **must** support Confidentiality and Integrity of workload resource utilization (RAM, CPU, Storage, Network I/O, cache, hardware offload) and restrict information sharing with only the workload owner (e.g., tenant). |  | |
+| sec.ci.007 | The Platform **must not** allow Memory Inspection by any actor other than the authorized actors for the Entity to which Memory is assigned (e.g., tenants owning the workload), for Lawful Inspection, and by secure monitoring services |  | Admin access must be carefully regulated |
+| sec.ci.008 | The NFVI **must** support tenant networks segregation |  | |
+
+<a name="7.11.4"></a>
+### 7.11.4. Workload Security
+
+| Ref | Requirement | Unit | Definition |
+|---|----|---|----|
+| sec.wl.001 | The Platform **must** support Workload placement policy |  | |
+| sec.wl.002 | The Platform **must** support operational security |  | |
+| sec.wl.003 | The Platform **must** support secure provision of workloads  |  | |
+| sec.wl.004 | The Platform **must** support Location assertion (for mandated in-country or location requirements) |  | |
+| sec.wl.005 | Production workloads **must** be separated from non-production workloads |  | |
+| sec.wl.006 | Workloads **must** be separable by sensitivity level (for example, financial versus external facing web server) |  | |
+| sec.wl.007 | Platform **must** verify VNF authenticity and integrity |  | |
+
+<a name="7.11.5"></a>
+### 7.11.5. Image Security
+
+| Ref | Requirement | Unit | Definition |
+|---|----|---|----|
+| sec.img.001 | Only images from trusted sources **must** be used |  | |
```

Comment:
```suggestion
| sec.img.001 | Images from untrusted sources **must not** be used |  | |
```
A slight re-wording that I think clarifies the intent.

Reply:
@kedmison Done - Thanks